resources/lang/en/response/index.php

```diff
@@ -149,7 +149,7 @@
             ],
             3 => [
                 'title' => 'Step 03: Get placed in a pool called “Ready to Allocate”',
-                'body' => 'After your reference check is done, you’ll be placed on a list of people who are ready to be sent to departments that have critical needs in the area(s) you applied to. You may get called up immediately or you may not get called at all. We’ll also let you know when you’re no longer needed in reserve, so you’re not left wondering. While you’re in this pool, please make sure to ((email the team)) if you have to withdraw your name for any reason. We want to make sure we only send the names of people currently available to departments in need.',
+                'body' => 'After your reference check is done, you’ll be placed on a list of people who are ready to be sent to departments that have critical needs in the area(s) you applied to. You may get called up immediately or you may not get called at all. We’ll also let you know when you’re no longer needed in reserve, so you’re not left wondering. While you’re in this pool, please make sure to <a href="mailto:Talent.Cloud-nuage.de.talents@tbs-sct.gc.ca" title="Send Talent Cloud an email." target="_blank">email us</a> if you have to withdraw your name for any reason. We want to make sure we only send the names of people currently available to departments in need.',
```

Comment:
It bother's me a little that we're using the config('mail.admin_adress') elsewhere but hardcoding the email here. I'll fix that with a wildcard.

Reply:
I didn't know you could do that. I'll remember it for next time!